backend/migration_tools/management/commands/create_v2_schema.py

```diff
@@ -0,0 +1,16 @@
+import os
+
+from django.core.management.base import BaseCommand
+from django.db import connection
+
+
+class Command(BaseCommand):
+    help = "Create v2 schema if it does not exist"
+
+    def handle(self, *args, **kwargs):
+        schema_name = os.getenv("V2_SCHEMA_NAME", "unstract_v2")
```

Comment:
@hari-kuriakose should we hardcode this schema name instead of using an ENV? 

Just because we don't want to change this accidentally.

Reply:
In case of a migration, we can add a check to ensure that migrations are not applied to the public schema.

Reply:
It updated according to the discussions here https://github.com/Zipstack/unstract/pull/538

Reply:
I see that we have went with the ENV instead of hard coding. If that is the final call feel free to resolve this comment.